molo/profiles/admin.py

```diff
@@ -26,3 +47,6 @@ def _mobile_number(self, obj, *args, **kwargs):
         if hasattr(obj, 'profile') and obj.profile.mobile_number:
             return obj.profile.mobile_number
         return ''
+
+    def _date_joined(self, obj, *args, **kwargs):
+        return obj.date_joined.date()
```

Comment:
Yes, let's keep the time. It'll help when you need to find out whether the user joined in the morning or the evening.